res/freedesktop/solvespace.desktop.in

```diff
@@ -3,6 +3,7 @@ Version=1.0
 Name=SolveSpace
 Comment=A parametric 2d/3d CAD
 Exec=${CMAKE_INSTALL_FULL_BINDIR}/solvespace
+MimeType=model/slvs
```

Comment:
Shouldn't this be something like `application/x-solvespace`? I think all MIME types not explicitly assigned by IANA should be in the `x-` namespace.

Reply:
You're right. I'll make the change.